Sources/MapboxMaps/Style/Style+Localization.swift

```diff
@@ -113,4 +113,19 @@ extension String {
                                               range: range,
                                               withTemplate: replacement)
     }
+
+    /// Updates string once using the first occurrence of a regex
+    /// - Parameters:
+    ///   - replacement: New string to replace the matched pattern
+    ///   - regex: The regex pattern that will be matched for replacement
+    internal mutating func updateOnceExpression(replacement: String, regex: NSRegularExpression) {
+        var range = NSRange(location: 0, length: self.count)
```

Comment:
we should be using NSString(self).length instead of self.count. They can produce different results under some circumstances.

I would actually be in favor of converting this entire extension to be an extension on NSString instead of String.

Reply:
we could also remove the line `NSRange(location: 0, length: self.count)` entirely, I wanted to use it as an initializer, but rangeOf is guaranteed to return a range either way, so it should be fine